### PR TITLE
Fix focus-tab --previous/next to ignore tab switcher order

### LINKED ISSUE #10070: The `nextTab` and `prevTab` actions do not work correctly when initiated via `wt.exe` or the command palette
In short, `wt focus-tab --next` and `--previous` don't do anything at all on my system, while `Next tab` and `Previous tab` via the command palette behave oddly, although I can't imagine that the latter is often used by anyone.

### Windows Terminal version (or Windows build number)

1.8.1032.0

#### Case 1:
Run `wt -w new new-tab ; new-tab ; focus-tab --next` (or `--previous`).

#### Case 2:
Run `wt -w new new-tab ; new-tab`.

In the new window, run `wt -w 0 focus-tab --next` (or `--previous`).

#### Case 3:
Run `wt -w new new-tab ; new-tab ; new-tab`.

In the new window, open the command palette (by default using <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>) and type in `Next tab` (or `Previous tab`) <kbd>Enter</kbd>.

#### Case 1:
The first tab is focused.

#### Case 2:
The first tab is focused.

#### Case 3:
The desired tab (according to the `tabSwitcherMode`) is focused.

#### Case 1:
The `focus-tab` subcommand is ignored; the second tab remains focused.

#### Case 2:
The `focus-tab` subcommand is ignored; the second tab remains focused.

Note that in the two cases above, `wt focus-tab --target 0` works correctly.

#### Case 3:
This depends on the `tabSwitcherMode`:

 * `mru` — the desired tab is focused for a split second, and then the originally-focused tab regains focus.
 * `inOrder` — the desired tab is focused for a split second, and then tab index 0 gains focus.
 * `disabled` — the desired tab is focused (there is no bug in this case).

----------

My apologies if I should have separated this into two issues.

Thanks!

### PR DESCRIPTION
**Pull request by Don-Vito**
When creating `startupAction` use `TabSwitcherMode::Disabled` in action args
to disable the tab switcher and prevent MRU logic to be applied.

Closes #10070

### CODE REVIEW

**Comment by Don-Vito** [comment]
Found no documentation to update: while -t is documented, -p, -n seem not to be. Am I missing something?

File: src/cascadia/TerminalApp/AppCommandlineArgs.cpp [hunk 1]
```diff
@@ -380,6 +380,9 @@ void AppCommandlineArgs::_buildFocusTabParser()
             else if (_focusNextTab || _focusPrevTab)
             {
                 focusTabAction.Action(_focusNextTab ? ShortcutAction::NextTab : ShortcutAction::PrevTab);
+                focusTabAction.Args(_focusNextTab ?
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/AppCommandlineArgs.cpp:383]:
```suggestion
                // GH#10070 - make sure to not use the MRU order when switching 
                // tabs on the commandline. That wouldn't make any sense!
                focusTabAction.Args(_focusNextTab ?
```

**Review by zadjii-msft — APPROVED** [review]
My only nit is adding a comment on why we need to do this. Otherwise, this is a very clever solution!

**Comment by zadjii-msft** [comment]
> Found no documentation to update: while -t is documented, -p, -n seem not to be. Am I missing something?

Nah, we just don't always remember to update docs when we add features. We try to, but sometimes we forget ¯\\\_(ツ)_/¯

**Review by leonMSFT — APPROVED** [review]
Thanks for this! 😄

**Review by DHowett — APPROVED** [review]
It's a beauty. Thanks!

**Comment by DHowett** [comment]
admin override: Not gonna spin the CI for a comment change

**Pull request merged**